examples/rinkeby.rs

```diff
@@ -0,0 +1,63 @@
+use ethcontract::transaction::Account;
+use ethsign::SecretKey;
+use std::env;
+use std::time::Duration;
+use web3::api::Web3;
+use web3::transports::WebSocket;
+use web3::types::H256;
+
+ethcontract::contract!("examples/truffle/build/contracts/DeployedContract.json");
+
+const RINKEBY_CHAIN_ID: u64 = 4;
+
+fn main() {
+    futures::executor::block_on(run());
+}
+
+async fn run() {
+    let account = {
+        let pk = env::var("PK").expect("PK is not set");
+        let raw_key: H256 = pk.parse().expect("invalid PK");
+        let key = SecretKey::from_raw(&raw_key[..]).expect("invalid PK");
```

Comment:
Is from_env_var some helper that would be useful to expose?

Reply:
Perhaps, but that would be something to add to `ethsign`. This is just a re-exported type: https://docs.rs/ethsign/0.7.3/ethsign/struct.SecretKey.html